.github/workflows/build-and-push-image.yml

```diff
@@ -173,3 +173,9 @@ jobs:
         with:
           name: screenshots-${{ needs.set-env.outputs.environment }}
           path: Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/screenshots
+          
+      - name: Report results
+        if: always()
+        run: npm run cy:notify -- --custom-text="Environment ${{ needs.set-env.outputs.environment }}, See more information https://github.com/DFE-Digital/prepare-academy-conversions/actions/runs/${{github.run_id}}"
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Have you added this secret?

Reply:
Yes it has been added to the repo. as   SLACK_WEBHOOK_UR. Thanks

Reply:
Has it been added to Terraform etc so it'll be in dev and test?

Reply:
I don't think we need it on Terraform, it just need to be on GitHub.